### ISSUE TITLE
export keys in formats other than ascii

### ISSUE THREAD

**Issue by ohadh** [mailvelope/mailvelope#537]
most apps i've run into will only import keys in pkcs12 format, none have accepted the .asc format exported by mailvelope. would be nice to be able to export the keys in other formats so we wouldn't need to create multiple keys for different apps

**Comment by FalcoGer** [comment]
I agree. This might help in the meanwhile: http://wiki.cacert.org/ConvertingPgpKeyToCertificate
